--- gtranslator-1.1.5/configure.in~	Fri Jul 23 13:32:13 2004
+++ gtranslator-1.1.5/configure.in	Fri Jul 23 13:32:13 2004
@@ -59,7 +59,7 @@
 GETTEXT_PACKAGE=gtranslator
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE(GETTEXT_PACKAGE,$GETTEXT_PACKAGE)
-ALL_LINGUAS="am az be ca cs da de el en_CA en_GB es et eu fa fi fr ga gl hr hu id is it ja ko lt lv nl nn no pa pl pt pt_BR ru sk sq sr sr@Latn sv ta tr th uk vi wa zh_CN zh_TW"
+ALL_LINGUAS="am az be ca cs da de el en_CA en_GB es et eu fa fi fr ga gl hr hu id is it ja ko lt lv nb nl nn pa pl pt pt_BR ru sk sq sr sr@Latn sv ta tr th uk vi wa zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 AC_CHECK_FUNC(bind_textdomain_codeset,,[AC_CHECK_LIB(intl,bind_textdomain_codeset)])
 
