--- gtranslator-1.1.7/configure.in.orig	2008-03-21 14:13:22.573701134 +0100
+++ gtranslator-1.1.7/configure.in	2008-03-21 14:14:02.070986578 +0100
@@ -122,7 +122,7 @@
 AC_CHECK_PROG(sc_present, scrollkeeper-config, yes, no)
 if test "z$sc_present" = "zyes" ; then
 	AC_MSG_CHECKING([for scrollkeeper])
-	_VERSION=`scrollkeeper-config --version|sed -e 's/\.//g'`
+	_VERSION=`scrollkeeper-config --version|sed -e 's/ .*$// ; s/\.//g'`
 	if test $_VERSION -ge 0 -o $_VERSION -ge 02; then
 		AC_MSG_RESULT(yes)
 		SC_STATEDIR=`scrollkeeper-config --pkglocalstatedir`
